.changeset/nasty-cobras-yawn.md

```diff
@@ -0,0 +1,6 @@
+---
+"examples": patch
```

Comment:
```suggestion
```

Only the sub-packages in the `packages/` folder should be tracked. I thought this thing would be ignored 🤔

Reply:
Realised this a bit too late. Fixed now.